packages/opentelemetry-cloud-trace-exporter/src/transform.ts

```diff
@@ -147,3 +147,35 @@ function valueToAttributeValue(
       return {};
   }
 }
+
+const HTTP_ATTRIBUTE_MAPPING: {[key: string]: string} = {
```

Comment:
The python impl you found was done most recently, and I think has the best mapping we can do right now. I need to get with the Cloud Trace folks to finalize these. Quite a few of the ones here don't appear in https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects.traces#get. Did they actually work?